diff --git a/src/decide.ts b/src/decide.ts
index 4220b7d..9bc7cfb 100644
--- a/src/decide.ts
+++ b/src/decide.ts
@@ -336,9 +336,6 @@ export async function assessCandidate(
 		if (!sourceDoesMatch(searchee.title, name)) {
 			return { decision: Decision.SOURCE_MISMATCH };
 		}
-		if (!releaseVersionDoesMatch(searchee.title, name)) {
-			return { decision: Decision.PROPER_REPACK_MISMATCH };
-		}
 		const size = metaOrCandidate.size;
 		if (size && !fuzzySizeDoesMatch(size, searchee)) {
 			return { decision: Decision.FUZZY_SIZE_MISMATCH };
